backend/app/routers/elasticsearch.py

```diff
@@ -23,3 +23,10 @@ async def search_dataset(request: Request):
     es = connect_elasticsearch()
     query = await request.body()
     return search_index(es, "dataset", query)
+
+
+@router.post("/_msearch")
+async def search_file_and_dataset(request: Request):
```

Comment:
can you rename this route `@router.post("/file,dataset/_msearch")`; that's what reactive search needed

Reply:
Done